jvm-libs/linea/besu-rlp-and-mappers/src/main/kotlin/linea/rlp/BlockRlpEncoderInterfaces.kt

```diff
@@ -0,0 +1,30 @@
+package linea.rlp
+
+import tech.pegasys.teku.infrastructure.async.SafeFuture
+
+interface BlockRlpEncoder<T> {
```

Comment:
These `BlockRlp*` don't actually care what to encode/decode, so they aren't specific to blocks or RLP. They just encode/decode 1 thing and list of things